generator/src/test/java/com/scottlogic/deg/generator/cucumber/engine/utils/CucumberGenerationConfigValidator.java

```diff
@@ -1,4 +1,4 @@
-package com.scottlogic.deg.generator.cucumber.utils;
+package com.scottlogic.deg.generator.cucumber.engine.utils;
```

Comment:
Why was this change made?
not all of these are necessarily related to the engine

Reply:
Not the generation engine, the testing engine (to separate the tests from the code that enables them). Probably `framework` would've been a better name?

Reply:
the tests are already separate because they should be in the features folder.

Can we revert this change? it is confusing having 2 things called engine.

Reply:
Happy to name it something else, but I want two things:

1) A directory that has all, and only, .feature files in it
2) For the aforementioned directory to be highly discoverable (ie, not hidden among the subfolders of the cucumber step definitions, etc)

Is there a mutually satisfying solution that enables these things? Eg, if I rename the `engine` directory to `glue` (seems to be the standard cucumber term) or `framework` or `testframework`?